src/components/shared/CheckboxField/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+type TextboxFieldProps = {
+  id: string;
+  label: string;
+  name: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  onBlur: () => void;
+  value: string;
+};
+
+const TextboxField = ({
```

Comment:
This component here is misnamed.

Also, is it worth making a component for a list of checkboxes? Seems like a common enough use case, since a list of checkboxes will likely share props.